Podfile

```diff
@@ -8,18 +8,12 @@ inhibit_all_warnings!
 
 app_ios_deployment_target = Gem::Version.new('16.0')
 
-def common_pods
-  pod 'google-cast-sdk-no-bluetooth', git: 'https://github.com/Automattic/google-cast'
-end
-
 target 'podcasts' do
   platform :ios, app_ios_deployment_target.version
-  common_pods
 end
 
 target 'PocketCastsTests' do
   platform :ios, app_ios_deployment_target.version
-  common_pods
 end
```

Comment:
@mokagio do you think we can get rid of these?

Reply:
@danielebogo 100%

Once this PR lands and we merge #3238 and #3239 , I'll rebase https://github.com/Automattic/pocket-casts-ios/pull/3241 to get all these changes and the whole of CocoaPods will be gone 🔥